File: server/src/main/java/com/fluffy/exam/application/request/PublishExamRequest.java
package com.fluffy.exam.application.request;

import com.fluffy.exam.application.request.question.QuestionRequest;
import com.fluffy.global.web.Accessor;
import java.util.List;

public record PublishExamRequest(
        Long examId,
        List<QuestionRequest> questions,
        Accessor accessor
) {
}

File: server/src/main/kotlin/com/fluffy/exam/application/request/PublishExamRequest.kt
package com.fluffy.exam.application.request

import com.fluffy.exam.application.request.question.QuestionRequest
import com.fluffy.global.web.Accessor

data class PublishExamRequest(
    val examId: Long,
    val questions: List<QuestionRequest>,
    val accessor: Accessor
)
